cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -25,9 +28,21 @@ type powershellScript struct {
 	envVars       []string
 }
 
+func checkInstalled(options tools.ExecOptions) (err error) {
+	return tools.ToolInPath(options.UserPwsh)
+}
+
 // Executes the specified powershell script
 // When interactive is true will attach to stdin, stdout & stderr
 func (bs *powershellScript) Execute(ctx context.Context, path string, options tools.ExecOptions) (exec.RunResult, error) {
+	if err := checkInstalled(options); err != nil {
+		return exec.RunResult{}, &internal.ErrorWithSuggestion{
+			Err: err,
+			Suggestion: fmt.Sprintf("PowerShell 7 is not installed or not in the path. To install PowerShell 7, visit %s",
```

Comment:
Sometimes including the developer name may also be handy -- feel free to decide if this makes it better or wose
```suggestion
			Suggestion: fmt.Sprintf("PowerShell 7 (pwsh.exe) is not installed or not in the path. To install PowerShell 7, visit %s",
```

Reply:
In linux or mac, it's `pwsh` over `pwsh.exe`